src/fhir/api.test.ts

```diff
@@ -3,12 +3,129 @@ import { createBundle } from "./api";
 import { initialValues, initialWithNoVariant } from "../components/reports/FormDefaults";
 import { Observation } from "fhir/r4";
 import { geneCoding } from "../code_systems/hgnc";
+import { Bundle } from "@smile-cdr/fhirts/dist/FHIR-R4/classes/bundle";
+import { BundleEntry } from "@smile-cdr/fhirts/dist/FHIR-R4/classes/bundleEntry";
+import { Patient } from "@smile-cdr/fhirts/dist/FHIR-R4/classes/models-r4";
 
 const fhir = new Fhir();
 
 const reportedGenes = [geneCoding("HGNC:4389", "GNA01")];
+const FHIR_URL = process.env.REACT_APP_FHIR_URL || "";
+
+const check = async (response: Response) => {
+  const r = await response.json();
+  if (!response.ok) {
+    console.error(r.body);
+    throw new Error(response.statusText);
+  }
+  console.debug(`check response: ${JSON.stringify(r, null, 2)}`);
+  return r;
+};
+
+const getPatients = async (identifier?: string): Promise<Bundle> => {
+  let url = `${FHIR_URL}/Patient`;
+  if (identifier) url = `${FHIR_URL}/Patient?identifier=${identifier}`;
+  const response = await fetch(url);
+  return await check(response);
+};
+
+const sendBundle = async (bundle: Bundle) => {
+  const sentBundle = await fetch(`${FHIR_URL}/`, {
+    method: "POST",
+    body: JSON.stringify(bundle),
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+  await new Promise((r) => setTimeout(r, 1500));
```

Comment:
worth trying to see if we remove this that this still works, I think we might not need it, ifso then we can remove the jest.setTimeout below as well

Reply:
pass/fail is iffy either way on local ://

Reply:
interesting, which tests?

Reply:
they all seem fine (for) now but I believe it was the `database is clear on setup` and `bundle creates patient`

Reply:
and what are the errors each of the tests? If they happen again, make sure to note them down